### DISCUSSION TITLE
Super Slicer Orginization

### DISCUSSION THREAD

**Discussion by abtblk** [supermerill/SuperSlicer#3939] [Polls]
So I have recently been printing many parts and I don't want to organize them because 1 I'm too lazy and 2 they won't align perfectly so I'll want to make them perfect. Should there be a "organize parts" feature?????

**Comment by jhlchu** [comment]
Is this a FR for a feature that is different than the "Arrange" already present in SS?

**Comment by supermerill** [comment]
If the Arrange button isn't up to the task, please submit an issue (with the feature request template)
